csharp/src/Drivers/Apache/Spark/SparkHttpConnection.cs

```diff
@@ -110,23 +108,22 @@ protected override void ValidateConnection()
             Properties.TryGetValue(SparkParameters.Path, out string? path);
             _ = new HttpClient()
             {
-                BaseAddress = GetBaseAddress(uri, hostName, path, port)
+                BaseAddress = GetBaseAddress(uri, hostName, path, port, Convert.ToBoolean(TlsOptions.IsSslEnabled))
```

Comment:
Unnecessary conversion.
```suggestion
                BaseAddress = GetBaseAddress(uri, hostName, path, port, TlsOptions.IsSslEnabled)
```